stock_picking_package_preparation/views/stock_picking_package_preparation_view.xml

```diff
@@ -4,7 +4,7 @@
         <field name="name">stock.picking.package.preparation.tree</field>
         <field name="model">stock.picking.package.preparation</field>
         <field name="arch" type="xml">
-            <tree colors="grey:state == 'cancel';" string="Package Preparations">
+            <tree decoration-info="state == 'cancel'" string="Package Preparations">
```

Comment:
I think that decoration-muted could be a better option. In previous version the module use color grey, so we can set the same color.
```suggestion
            <tree decoration-muted="state == 'cancel'" string="Package Preparations">
```